Modules/Filtering/FFT/src/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 set(ITKFFT_SRCS
 itkComplexToComplexFFTImageFilter.cxx
-itkVnlFFTImageFilterInitFactory.cxx)
+itkVnlFFTImageFilterInitFactory.cxx
+itkFFTWFFTImageFilterInitFactory.cxx)
 
 if(ITK_USE_FFTWF OR ITK_USE_FFTWD AND NOT ITK_USE_CUFFTW)
   list(APPEND ITKFFT_SRCS itkFFTWGlobalConfiguration.cxx)
-  list(APPEND ITKFFT_SRCS itkFFTWFFTImageFilterInitFactory.cxx)
```

Comment:
We can preserve conditions on adding `itkFFTWFFTImageFilterInitFactory.cxx` to sources while fixing CUFFTW builds with something like this:

```sh
if(ITK_USE_FFTWF OR ITK_USE_FFTWD)
  list(APPEND ITKFFT_SRCS itkFFTWFFTImageFilterInitFactory.cxx)
  if(NOT ITK_USE_CUFFTW)
    list(APPEND ITKFFT_SRCS itkFFTWGlobalConfiguration.cxx)
  endif()
endif()
```